### PR TITLE
Display username while removing the user from the group

### PR DESCRIPTION
**Pull request by kanavnarula**
This change is to display username (for the confirmation) while removing the user from the group.

**Before**:
image

**After**:
image

## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Comment by kanavnarula** [comment]
@david-leifker - Does it make sense to keep double quotes around the username in remove group modal capture above ?

**Review by jjoyce0510 — CHANGES_REQUESTED** [review]
Hi @kanavnarula!

Thanks for the contribution. In general, the idea makes sense . The following comment must be addressed before merging this one in: 

-  We never should use the "URN" of the user to display their name, as we are doing in this PR. Instead, we should use the display name of the entity which can be obtained via entityRegistry.getDisplayName(EntityType.CorpUser, entity) 

Thanks
John

**Comment by kanavnarula** [comment]
> Hi @kanavnarula!
> 
> Thanks for the contribution. In general, the idea makes sense . The following comment must be addressed before merging this one in:
> 
> * We never should use the "URN" of the user to display their name, as we are doing in this PR. Instead, we should use the display name of the entity which can be obtained via entityRegistry.getDisplayName(EntityType.CorpUser, entity)
> 
> Thanks John

Hi @jjoyce0510,

Thanks for looking in the PR. URN was coming as string to the function so could not use entityRegistry.getDisplayName(EntityType.CorpUser, entity).

I have updated the function signature to accept CorpUser entity instead of string. Please check and suggest if these changes are looking fine.

Thanks 
Kanav Narula

**Comment by rtekal** [comment]
@jjoyce0510 When you get a chance, please review this PR.  Thanks in advance.

**Comment by kanavnarula** [comment]
@jjoyce0510 - Did you get chance to check the PR changes.

**Comment by kanavnarula** [comment]
@jjoyce0510 @david-leifker - Please review when you get chance. Thanks in advance!

**Review by david-leifker — APPROVED** [review]
Seems like it addressed the comments. Going to merge. Thanks!

File: datahub-web-react/src/app/entity/group/GroupMembers.tsx [hunk 1]
```diff
@@ -137,12 +137,13 @@ export default function GroupMembers({ urn, pageSize, isExternalGroup, onChangeM
         }, 3000);
     };
 
-    const onRemoveMember = (memberUrn: string) => {
+    const onRemoveMember = (memberEntity: CorpUser) => {
+        const memberName = memberEntity?.properties?.displayName || undefined;
```
Comment (jjoyce0510) [inline datahub-web-react/src/app/entity/group/GroupMembers.tsx:141]: please use entityRegistry.getDisplayName(EntityType.CorpUser, memberEntity)

Comment (kanavnarula) [reply datahub-web-react/src/app/entity/group/GroupMembers.tsx]: @jjoyce0510 - Logic updated to pull display name using above command. Thanks!

**Comment by kanavnarula** [comment]
@jjoyce0510 @david-leifker - Closing this PR as I messed up by adding other file changes related to another contribution.
New PR created - https://github.com/datahub-project/datahub/pull/11706

**Pull request closed without merging**